src/opcuabridge/currenttime.h

```diff
@@ -0,0 +1,62 @@
+#ifndef OPCUABRIDGE_CURRENTTIME_H_
+#define OPCUABRIDGE_CURRENTTIME_H_
+
+#include "signature.h"
+#include "signed.h"
+
+#include "common.h"
+
+namespace opcuabridge {
+class CurrentTime {
+ public:
+  CurrentTime() {}
+  virtual ~CurrentTime() {}
+ public:
```

Comment:
For consistency with the rest of the codebase, can you include the public:/protected:/private: keyword just once, please?  https://google.github.io/styleguide/cppguide.html#Declaration_Order